controllers/controller_test.go

```diff
@@ -230,24 +233,26 @@ var _ = Describe("snr Controller", func() {
 
 				verifyNodeBackup()
 
-				verifyFinalizerExists()
-
-				verifyNoExecuteTaintExist()
-
 				verifyNoWatchdogFood()
 
 				verifySelfNodeRemediationPodDoesntExist()
 
 				verifyVaDeleted(vaName)
 
+				verifyFinalizerExists()
+
+				verifyNoExecuteTaintExist()
+
+				deleteSNR(snr)
+				isDeleteSNR = false
```

Comment:
the name `isDeleteSNR` is confusing... 🤔

Reply:
>Maybe in this case that var isn't needed at all, just ignore isNotFound error when trying to delete the CR?

This will affect almost all the tests in this file making them less robust IMO, so I'd prefer to avoid that.

Regarding the var name personally I'm fine with that name, but I'm defiantly open to other suggestions :smile:

Reply:
anything which is meaningful without reading the code would be fine.
for `isDeleteSNR` I needed to read the code...
maybe `isSNRDeleted` (would switch true / false meaning)
or `needsSNRDeletion` or `deleteSNRInAfterEach`
...

Reply:
>anything which is meaningful without reading the code would be fine.

I guess that is subjective, for me `isDeleteSNR` fits this criteria perfectly,
This is how I read it:
is - an indication that this is a flag (or whether or not to preform the action)
Delete SNR - the action that should be done if the flag is true.

I'll switch to `isSNRneedsDeletion` I'm hoping it's something we can have consensus on  :crossed_fingers: